python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/conftest.py

```diff
@@ -232,168 +234,180 @@ def get_sample_connection_details(succeeded_at: str, failed_at: str) -> Mapping[
 
 # Taken from Fivetran API documentation
 # https://fivetran.com/docs/rest-api/api-reference/connector-schema/connector-schema-config
-SAMPLE_SCHEMA_CONFIG_FOR_CONNECTOR = {
-    "code": "Success",
-    "message": "Operation performed.",
-    "data": {
-        "enable_new_by_default": True,
-        "schemas": {
-            "property1": {
-                "name_in_destination": "schema_name_in_destination_1",
-                "enabled": True,
-                "tables": {
-                    "property1": {
-                        "sync_mode": "SOFT_DELETE",
-                        "name_in_destination": "table_name_in_destination_1",
-                        "enabled": True,
-                        "columns": {
-                            "property1": {
-                                "name_in_destination": "column_name_in_destination_1",
-                                "enabled": True,
-                                "hashed": False,
-                                "enabled_patch_settings": {
-                                    "allowed": False,
-                                    "reason": "...",
-                                    "reason_code": "SYSTEM_COLUMN",
+# The sample is parameterized to test the sync and poll materialization method
+def get_sample_schema_config_for_connector(table_name: str) -> Mapping[str, Any]:
+    return {
```

Comment:
No changes in the dictionary other than the `table_name` variable - marked as new by git because of the new indentation.